openedx_caliper_tracking/kafka_settings.py

```diff
@@ -0,0 +1,30 @@
+"""
+Contains class for kafka settings.
+"""
+from django.conf import settings
+
+
+class KafkaSettings:
+    """
+    Contains methods to get kafka settings.
+    """
+
+    @staticmethod
+    def get_maximum_retries():
+        if hasattr(settings, 'CALIPER_KAFKA_SETTINGS'):
+            return settings.CALIPER_KAFKA_SETTINGS.get('MAXIMUM_RETRIES')
```

Comment:
No need for that, because if the `if` block isn't executed `None` would be returned by default.

Take a look at example below.

![image](https://user-images.githubusercontent.com/42294172/67659030-306a1e80-f97d-11e9-8ab6-26ddfbcf3963.png)

Reply:
cool